utilities/pipelines/dependencies/Microsoft.Authorization/policyAssignments/parameters/parameters.json

```diff
@@ -0,0 +1,18 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "name": {
+            "value": "sxx-assignment-loc-rg"
```

Comment:
can we change the name to contain 'adp', to align with other dependency resources? Maybe something like `adp-sxx-polassign-loc-rg`. It also meets the name length requirement.

Reply:
good point and done